testcases/MachineConfig.py

```diff
@@ -368,15 +385,31 @@ def ValidateCEC_Setup(self):
             if int(current_lmb[0]) == int(self.lmb_size):
                 log.info("System booted with LMB %s" % self.lmb_size)
             else:
-                return "Failed to boot with LMB %s" % self.lmb_size
+                self.cec_dict['lmb_cec'] = self.lmb_size
+        if self.num_hugepages:
+            self.current_hgpg = self.cv_HMC.get_16gb_hugepage_size()
+            if int(self.current_hgpg[0]) != 0:
+                log.info("System configured with %s 16gb Hugepage" % self.current_hgpg[0])
+                self.cv_HMC.poweroff_lpar()
```

Comment:
suppose if cec already has 16gb hugepages configured then I need to allocate hugepages to the lpar from HMC.. These changes will apply if we do reboot of the system. So that's why am powering off the lpar setting the attributes and powering on the lpar